src/Agent/NewRelic/Agent/Extensions/NewRelic.Agent.Extensions/Api/InternalApi.cs

```diff
@@ -263,9 +263,9 @@ public static void IgnoreApdex()
         ///   &lt;body&gt;
         ///   ...
         /// </code></example>
-        public static string GetBrowserTimingHeader()
```

Comment:
As the result of my comment above, we like to keep 2 overloading methods of the `GetBrowserTimingHeader`.